indico/MaKaC/webinterface/pages/main.py

```diff
@@ -48,12 +43,15 @@ def _display( self, params ):
         body = WMainBase(self._getBody(params), self._timezone, self._getNavigationDrawer(),
                          isFrontPage=self._isFrontPage(),
                          isRoomBooking=self._isRoomBooking(),
-                         sideMenu=sideMenu).getHTML({"subArea": self._getSiteArea()})
+                         sideMenu=self._getSideMenu()).getHTML({"subArea": self._getSiteArea()})
 
         return self._applyDecoration(body)
 
     def _getBody(self, params):
-        return _("nothing yet")
+        return ''
```

Comment:
`NotImplementedError` - `NotImplemented` is meant for places where you return it (e.g. comparison functions)